Makefile

```diff
@@ -196,46 +204,33 @@ reset_owner:
+	${DOCKER_COMPOSE_RUN} run --rm backend perl /opt/product-opener/scripts/create_mongodb_indexes.pl
 
 refresh_product_tags:
 	@echo "🥫 Refreshing product data cached in Postgres …"
 	${DOCKER_COMPOSE} run --rm backend perl /opt/product-opener/scripts/refresh_postgres.pl ${from}
 
-import_sample_data:
+import_sample_data:run_deps
 	@ if [[ "${PRODUCT_OPENER_FLAVOR_SHORT}" = "off" &&  "${PRODUCERS_PLATFORM}" != "1" ]]; then \
    		echo "🥫 Importing sample data (~200 products) into MongoDB …"; \
-		${DOCKER_COMPOSE} run --rm backend bash /opt/product-opener/scripts/import_sample_data.sh; \
+		${DOCKER_COMPOSE_RUN} run --rm backend bash /opt/product-opener/scripts/import_sample_data.sh; \
 	else \
 	 	echo "🥫 Not importing sample data into MongoDB (only for po_off project)"; \
 	fi
 	
-import_more_sample_data:
+import_more_sample_data:run_deps
 	@echo "🥫 Importing sample data (~2000 products) into MongoDB …"
-	${DOCKER_COMPOSE} run --rm backend bash /opt/product-opener/scripts/import_more_sample_data.sh
+	${DOCKER_COMPOSE_RUN} run --rm backend bash /opt/product-opener/scripts/import_more_sample_data.sh
+
+refresh_mongodb:run_deps
+	@echo "🥫 Refreshing mongoDB from product files …"
+	${DOCKER_COMPOSE_RUN} run --rm backend perl /opt/product-opener/scripts/update_all_products_from_dir_in_mongodb.pl
 
 # this command is used to import data on the mongodb used on staging environment
-import_prod_data:
-	@echo "🥫 Importing production data (~2M products) into MongoDB …"
-	@echo "🥫 This might take up to 10 mn, so feel free to grab a coffee!"
-	@echo "🥫 Removing old archive in case you have one"
-	( rm -f ./html/data/openfoodfacts-mongodbdump.gz || true ) && ( rm -f ./html/data/gz-sha256sum || true )
-	@echo "🥫 Downloading full MongoDB dump from production …"
-# verify we got sufficient space, NEEDED is in octet, LEFT in ko, we normalize to MB and NEEDED is multiplied by two (because it also will be imported)
-	NEEDED=$$(curl -s --head https://static.openfoodfacts.org/data/openfoodfacts-mongodbdump.gz|grep -i content-length: |cut -d ":" -f 2|tr -d " \r\n\t"); \
-	  LEFT=$$(df . -k --output=avail |tail -n 1); \
-	  NEEDED=$$(($$NEEDED/1048576 * 2)); \
-	  LEFT=$$(($$LEFT/1024)); \
-	  if [[ $$LEFT -lt $$NEEDED ]]; then >&2 echo "NOT ENOUGH SPACE LEFT ON DEVICE: $$NEEDED MB > $$LEFT MB"; exit 1; fi
-	wget --no-verbose https://static.openfoodfacts.org/data/openfoodfacts-mongodbdump.gz -P ./html/data
-	wget --no-verbose https://static.openfoodfacts.org/data/gz-sha256sum -P ./html/data
-	cd ./html/data && sha256sum --check gz-sha256sum
-	@echo "🥫 Restoring the MongoDB dump …"
-	${DOCKER_COMPOSE} exec -T mongodb //bin/sh -c "cd /data/db && mongorestore --quiet --drop --gzip --archive=/import/openfoodfacts-mongodbdump.gz"
-	rm html/data/openfoodfacts-mongodbdump.gz && rm html/data/gz-sha256sum
+import_prod_data: run_deps
+	@cd ${DEPS_DIR}/openfoodfacts-shared-services && $(MAKE) -e import_prod_data
```

Comment:
You can remove it from this file… it's basically used by the daily task, it concerns mongodb only and it does not make much sense keeping it here (it belongs to shared-net repository)

Reply:
I kept it as I have used it myself before to get all data locally and it is also referenced in docs.